Packages/com.unity.inputsystem/InputSystem/Controls/InputControlExtensions.cs

```diff
@@ -912,22 +912,24 @@ internal static string BuildPath(this InputControl control, string deviceLayout,
             var device = control.device;
 
             builder.Append('<');
-            builder.Append(deviceLayout);
+            builder.Append(deviceLayout.Replace("\\", "\\\\").Replace(">", "\\>"));
```

Comment:
Curious why we don't use our own StringHelper.Escape method for all this escaping?

Reply:
Fixed